app/src/test/java/shop/woowasap/domain/product/DomainFixture.java

```diff
@@ -0,0 +1,19 @@
+package shop.woowasap.domain.product;
```

Comment:
test fixture와 validator 클래스를 support 패키지로 빼는건 어떤가요?

```suggestion
package shop.woowasap.domain.product.support.validator;
```

Reply:
오 좋습니다! test를 support해주는 static 메서드를 한 곳에 모으니 깔끔해졌네요!